lib/produce/service.rb

```diff
@@ -0,0 +1,205 @@
+      end
+
+      if options.data_protection
+        case options.data_protection
+          when "complete"
+            app.update_service(Spaceship.app_service.data_protection.complete)
+            enabled += 1
+            Helper.log.info "Enabled Data Protection: Complete"
+          when "unlessopen"
+            app.update_service(Spaceship.app_service.data_proection.unless_open)
+            enabled += 1
+            Helper.log.info "Enabled Data Protection: Unless Open"
+          when "untilfirstauth"
+            app.update_service(Spaceship.app_service.data_protection.until_first_auth)
+            enabled += 1
+            Helper.log.info "Enabled Data Protection: Until First Authentication"
+        end
+      end
+
+      if options.healthkit
+        app.update_service(Spaceship.app_service.health_kit.on)
+        enabled += 1
+        Helper.log.info "Enabled HealthKit"
+      end
+
+      if options.homekit
+        app.update_service(Spaceship.app_service.home_kit.on)
+        enabled += 1
+        Helper.log.info "Enabled HomeKit"
+      end
+
+      if options.wireless_conf
+        app.update_service(Spaceship.app_service.wireless_accessory.on)
+        enabled += 1
+        Helper.log.info "Enable Wireless Accessory Configuration"
+      end
+
+      if options.icloud
+        case options.icloud
+        when "legacy"
+          app.update_service(Spaceship.app_service.icloud.on)
+          app.update_service(Spaceship.app_service.cloud_kit.xcode5_compatible)
+          enabled += 1
+          Helper.log.info "Enabled iCloud: CloudKit"
+        when "cloudkit"
+          app.update_service(Spaceship.app_service.icloud.on)
+          app.update_service(Spaceship.app_service.cloud_kit.cloud_kit)
+          enabled += 1
+          Helper.log.info "Enabled iCloud: CloudKit"
+        end
+      end
+
+      if options.inter_app_audio
+        app.update_service(Spaceship.app_service.inter_app_audio.on)
+        enabled += 1
+        Helper.log.info "Enabled Inter-App Audio"
+      end
+
+      if options.passbook
+        app.update_service(Spaceship.app_service.passbook.on)
+        enabled += 1
+        Helper.log.info "Enabled Passbook"
+      end
+
+      if options.push_notification
+        app.update_service(Spaceship.app_service.push_notification.on)
+        enabled += 1
+        Helper.log.info "Enabled Push Notifications"
+      end
+
+      if options.vpn_conf
+        app.update_service(Spaceship.app_service.vpn_configuration.on)
+        enabled += 1
+        Helper.log.info "Enabled VPN Configuration"
+      end
+
+      Helper.log.info "Done! Enabled #{enabled} services.".green
+    end
+
+    def disable(options, args)
+      Helper.log.info "Starting login"
+      Spaceship.login(Produce.config[:username], nil)
+      Spaceship.select_team
+      Helper.log.info "Successfully logged in"
+
+      app = Spaceship.app.find(Produce.config[:app_identifier].to_s)
+
+      if app.nil?
```

Comment:
I've seen this code above, let's move that out into a function:

```ruby
def app
      Helper.log.info "Starting login"
      Spaceship.login(Produce.config[:username], nil)
      Spaceship.select_team
      Helper.log.info "Successfully logged in"
      app = Spaceship.app.find(Produce.config[:app_identifier].to_s)
end
```
which we can use for both methods